canonical-websites/www.ubuntu.com
# Add new developer terms and archived the old one.

## Done

Added a new developer terms and archived the old one on a dated page.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions)
- Check the page against the [copydoc](https://docs.google.com/document/d/1I0wumuzwywfKjiKC4YmQ-9h9O3ZvCaBTe2CIIyoEtJQ/edit#).
- Click the link in "Older versions" and confirm that the old version is correctly archived by date.


## Issue / Card

Fixes #1915
